src/javascript/app/Constants/contract.js

```diff
@@ -123,5 +123,56 @@ const getContractConfig = is_high_low => ({
     ...getUnsupportedContracts(),
 });
 
+export const getSupportedContracts = is_high_low => ({
+    CALL: {
+        name    : is_high_low ?  localize('Higher') : localize('Rise'),
+        position: 'top',
+    },
+    PUT: {
+        name    : is_high_low ? localize('Lower') : localize('Fall'),
+        position: 'bottom',
+    },
+    CALLE: {
+        name    : localize('Rise'),
+        position: 'top',
+    },
+    PUTE: {
+        name    : localize('Fall'),
+        position: 'bottom',
+    },
+    DIGITMATCH: {
+        name    : localize('Matches'),
+        position: 'top',
+    },
+    DIGITDIFF: {
+        name    : localize('Differs'),
+        position: 'bottom',
+    },
+    DIGITEVEN: {
+        name    : localize('Even'),
+        position: 'top',
+    },
+    DIGITODD: {
+        name    : localize('Odd'),
+        position: 'bottom',
+    },
+    DIGITOVER: {
+        name    : localize('Over'),
+        position: 'top',
+    },
+    DIGITUNDER: {
+        name    : localize('Under'),
+        position: 'bottom',
+    },
+    ONETOUCH: {
+        name    : localize('Touch'),
+        position: 'top',
+    },
+    NOTOUCH: {
+        name    : localize('No Touch'),
+        position: 'bottom',
+    },
+});
```

Comment:
as discussed please remove these from contract config and import this object in it instead.